static/app/views/performance/contexts/utils.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
Should this file exist within `static/app/utils/...` instead?

Reply:
Definitely, good call.